docs/product/components/partials/_topbar.html

```diff
@@ -0,0 +1,151 @@
+            <form id="search" class="s-topbar--searchbar" autocomplete="off">
+                {% unless loggedOut %}
+                    <div class="s-select">
+                        <select>
+                            <option selected="selected">
+                                All
+                            </option>
+                            <option>
+                                Public
+                            </option>
+                            <option>
+                                Private Team 1
+                            </option>
+                        </select>
+                    </div>
+                {% endunless %}
+                <div class="s-topbar--searchbar--input-group">
+                    <input type="text" placeholder="Search…" value="" autocomplete="off" class="s-input s-input__search" />
+                    {% icon "Search", "s-input-icon s-input-icon__search" %}
+                </div>
+            </form>
+        {% endunless %}
+
+        {% if loggedOut %}
+            <ol class="s-topbar--ctas">
+                <li>
+                    <a href="#" class="s-topbar--item s-btn s-btn__filled">Log in</a>
+                </li>
+                <li>
+                    <a href="#" class="s-topbar--item s-btn s-btn__primary">Sign up</a>
+                </li>
+            </ol>
+        {% else %}
+            <ol class="s-topbar--content">
+                <li>
+                    <button
+                        class="s-topbar--item s-btn s-btn__icon s-btn__muted d-none sm:d-inline-flex js-search-button"
+                        type="button">
+                            {% icon "Search" %}
+                    </button>
+                </li>
+                {% if isMod %}
+                    <li>
+                        <a href="#" class="s-topbar--item" title="Moderator inbox">
+                            {% icon "Moderator" %}
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="s-topbar--item" title="8 posts flagged for moderator attention">
+                            <span class="s-badge s-badge__bounty">8</span>
+                        </a>
+                    </li>
+                {% endif %}
+                <li>
+                    <a href="#" class="s-topbar--item s-user-card s-user-card__small">
+                        <span class="s-avatar s-avatar__24 s-user-card--avatar">
+                            <img class="s-avatar--image" src="{{ "/assets/img/anonymous-user.svg" | relative_url }}" />
+                            <span class="v-visible-sr">John Doe</span>
+                        </span>
+                        <div class="s-user-card--info">
+                            <ul class="s-user-card--awards">
+                                <li class="s-user-card--rep">3,145</li>
+                                <li class="s-award-bling s-award-bling__gold">3</li>
```

Comment:
Speaking of interdependencies, we won't be able to go live with this until we add custom CSS for bling. Otherwise this'll be a regression across the network. Not news to you of course, but noting it here for when we both forget lol.

Reply:
On Core I have this working / looking ok with the current markup that we're using for the bling. I want the Stacks docs to be 100% Stacks, but the topbar component is flexible enough in this aspect that we have quite a bit of wiggle room as to what we put inside the `s-topbar--item`s.